README.md

```diff
@@ -148,28 +153,78 @@ One may specify multiple data locations by a list of locations as long as they h
   ```
   location: ["./data/sample.parquet", "https://d37ci6vzurychx.cloudfront.net/trip-data/yellow_tripdata_2022-01.parquet"]
   ```
-- `using`: string. Required.  
-One of `parquet`, `spark_sql` and `delta`, or the custom file format specified by the
-loader function in `steps/ingest.py` (ex. `csv`).
 
 
-- `loader_method`: string. Optional.  
-Method name of the custom loader function from `steps/ingest.py`.  
-<u>Example</u>: 
+</details>
+
+<details>
+<summary><strong><u>Using: "delta"</u></strong></summary>
+
+- `location`: string. Required.  
+Dataset locations or the catalog path
```

Comment:
Expand on the explanation here: `A path pointing to the Delta table location or a catalog path (when running on Databricks) of the format "catalog.schema.table", for more details on the table format, see [here](https://docs.databricks.com/data-governance/unity-catalog/index.html)`

As a side note, can you confirm that this works for both UC format `catalog.schema.table` and regular Hive format `schema.table`?

Reply:
Yes I did try it out a while back. It did work